_episodes/40.md

```diff
@@ -0,0 +1,495 @@
+format will be a great extension for those using Iceberg tables as it will
+enable better query plans in Trino at the file level.
+
+### pyIceberg
+
+Another exciting development is the 
+[pyIceberg library](https://github.com/apache/iceberg/tree/master/python) that
+enables users to read their data directly from Iceberg into native python data
+structures.
+
+### Trino Iceberg connector updates
+
+* [`MERGE`]({{site.url}}/docs/current/sql/merge) ([PR](https://github.com/trinodb/trino/pull/7933))
+* [`UPDATE`]({{site.url}}/docs/current/sql/update) ([PR](https://github.com/trinodb/trino/pull/12026))
+* [`DELETE`]({{site.url}}/docs/current/sql/delete) ([PR](https://github.com/trinodb/trino/pull/11886))
+* Time travel ([PR](https://github.com/trinodb/trino/pull/10258)) was initially 
+released in 
+[version 385]({{site.url}}docs/current/release/release-385.html#iceberg-connector),
+the `@` syntax for snapshots/time travel
+[was deprecated](https://github.com/trinodb/trino/pull/10768) in 
+[version 387]({{site.url}}docs/current/release/release-387.html#iceberg-connector),
+and there were two bug fixes for this feature in versions 
+[386]({{site.url}}docs/current/release/release-386.html#iceberg-connector) and
+[388]({{site.url}}docs/current/release/release-388.html#iceberg-connector).
+* [Partition migration]({{site.url}}/docs/current/connector/iceberg.html#alter-table-set-properties)
+([PR](https://github.com/trinodb/trino/pull/12259))
+While Trino was able to read tables with these migrations applied by other query
+engines, this feature allows Trino to write these changes.
+* The following three features are table maintenance commands.
+    * [`optimize`]({{site.url}}/docs/current/connector/iceberg.html#optimize)
+([PR](https://github.com/trinodb/trino/pull/10497)) which is the equivalent to
+the Spark SQL 
+[rewrite_data_files](https://iceberg.apache.org/docs/latest/spark-procedures/#rewrite_data_files).
+    * [`expire_snapshots`]({{site.url}}/docs/current/connector/iceberg.html#expire-snapshots)
+([PR](https://github.com/trinodb/trino/pull/10810)) and uses the equivalent name
+in Spark.
+    * [`remove_orphan_files`]({{site.url}}/docs/current/connector/iceberg.html#remove-orphan-files)
+([PR](https://github.com/trinodb/trino/pull/10810)) and uses the equivalent name
+in Spark.
+* Iceberg v2 support ([PR1](https://github.com/trinodb/trino/pull/11880), [PR2](https://github.com/trinodb/trino/pull/12351), [PR3](https://github.com/trinodb/trino/pull/12749), [PR4](https://github.com/trinodb/trino/pull/11642), [PR5](https://github.com/trinodb/trino/pull/9881), and many more...)
+
+
+Almost every release, there is some sort of performance improvement around
+many use cases involving [planning](https://github.com/trinodb/trino/pull/13636),
+[pushdown](https://github.com/trinodb/trino/pull/13395), 
+[avoiding excessive scanning](https://github.com/trinodb/trino/pull/7367), and
+much more.  A lot of the common use cases have been covered but please 
```

Comment:
Very long and winded ..

Maybe just

If you experience an issue, open an issue

Or we can even leave it out..